.changeset/rich-jars-jog.md

```diff
@@ -0,0 +1,5 @@
+---
+"@khanacademy/wonder-blocks-button": minor
```

Comment:
**suggestion:** It's better to treat this as a breaking change as it is changing the existing API (`icon` => `startIcon`).

For reference: https://khanacademy.atlassian.net/wiki/spaces/WB/pages/2217640195/Versioning+and+releasing+Wonder+Blocks+packages#Major-changes-1.0.0

Reply:
Ah yeah, that's a good point